proto/ca_service.proto

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package proto;
+
+// Service definition of IstioCAService that can dynamically verify the CSR for
+// service identities for Istio services running on various platforms and
+// approve or disapprove the request after verifying the Node Agent credentials
+// provided as a part of the Certificte Signing Request.
+service IstioClusterCAService {
+
```

Comment:
Cluster is more a logical concept than a physical concept. The naming is to provide the intended behavior of the module rather than enforce constraints. You can look at https://docs.google.com/document/d/1ab_f7OnsuQ3Q8C-Al6yHr1pFNv_Z8BtfovzYEU5dII4/edit to see that a cluster is being using to define an Istio mesh on a Hybrid cloud.